pkg/clustermanager/cluster_manager.go

```diff
@@ -123,7 +123,7 @@ type Networking interface {
 type AwsIamAuth interface {
 	GenerateManifest(clusterSpec *cluster.Spec) ([]byte, error)
 	GenerateManifestForUpgrade(clusterSpec *cluster.Spec) ([]byte, error)
-	GenerateCertKeyPairSecret() ([]byte, error)
+	GenerateCertKeyPairSecret(bootstrapClusterName string) ([]byte, error)
```

Comment:
let's call this variable name 'clusterName' for clarity. It could either take workload or bootstrap cluster names

Reply:
Actually I named it this because we should always pass in the bootstrap cluster name here, either KinD or the management cluster.
I could change it and add a comment instead.